### PR TITLE
more fixing for the tests

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this opm-common=858 please

**Comment by GitPaean** [comment]
jenkins build this opm-common=858 please

File: opm/core/wells/WellsGroup.cpp [hunk 1]
```diff
@@ -1485,6 +1485,7 @@ namespace Opm
                 case Phase::POLYMER: OPM_THROW(std::invalid_argument, "Polymer injector is not supported.");
                 case Phase::ENERGY:  OPM_THROW(std::invalid_argument, "Energy injector is not supported.");
                 case Phase::POLYMW:  OPM_THROW(std::invalid_argument, "PolyMW injector is not supported.");
+                case Phase::FOAM:    OPM_THROW(std::invalid_argument, "Foam injector is not supported.");
             }
             OPM_THROW(std::logic_error, "Invalid state." );
```
Comment (GitPaean) [inline opm/core/wells/WellsGroup.cpp]: `"Invalid state."` is a very bad message here.  Probably we should add a default for the switch.  We should not address it in this PR though.

Comment (bska) [reply opm/core/wells/WellsGroup.cpp]: > Probably we should add a default for the switch.

Preferably not.  That will hide a useful compiler diagnostic that some enum values are not handled.

**Review by atgeirr — APPROVED** [review]
Fine, will merge.

File: opm/core/wells/WellsGroup.cpp (hunk 1 above)
Comment (GitPaean) [reply opm/core/wells/WellsGroup.cpp]: Does it need to be a `enum class` or it will work for the usual `enum`?

**Pull request merged**